dashboard/app/helpers/levels_helper.rb

```diff
@@ -449,6 +449,7 @@ def blockly_options
     if l.should_localize?
       level_options['markdownInstructions'] = l.localized_markdown_instructions unless l.localized_markdown_instructions.nil?
       level_options['failureMessageOverride'] = l.localized_failure_message_override unless l.localized_failure_message_override.nil?
+      level_options['toolbox'] = l.localized_toolbox_blocks unless l.localized_toolbox_blocks.nil?
```

Comment:
This pattern will call `l. localized_toolbox_blocks` twice, without caching the result.

Reply:
Yeah, this is not a great pattern we've gotten ourselves into. Fix here https://github.com/code-dot-org/code-dot-org/pull/22369/commits/cd364ff3a4b560d702cfe88a5ac438cee1ea0dc8 PTAL

Reply:
Looks good 👍